web/markdown/rules.react.js

```diff
@@ -47,8 +71,105 @@ function linkRules(): MarkdownRuleSpec {
 
 // function will contain additional rules for message formatting
 function markdownRules(): MarkdownRuleSpec {
+  const linkMarkdownRules = linkRules();
+
+  const simpleMarkdownRules = {
+    ...linkMarkdownRules.simpleMarkdownRules,
+    autolink: SimpleMarkdown.defaultRules.autolink,
+    link: {
+      ...linkMarkdownRules.simpleMarkdownRules.link,
+      match: SimpleMarkdown.defaultRules.link.match,
+    },
+    blockQuote: {
+      ...SimpleMarkdown.defaultRules.blockQuote,
+      // match end of blockQuote by either \n\n or end of string
+      match: SimpleMarkdown.blockRegex(/^( *>[^\n]+(\n[^\n]+)*\n*)+(\n{2,}|$)/),
```

Comment:
But in case of `blockQuote`, isn't two newlines at the end best way to catch end of quote? If we want blockQuote and normal text after in one message for example, but also we want to have option for blockQuote to be multiline

Reply:
So what happens with just a single newline? Like this:

```
> Block quote
Text
```